specification.md

```diff
@@ -121,6 +130,7 @@ To sign:
 -   BODY **must** be an object type (`{...}`).
 -   Serialize BODY as [Canonical JSON]; call this SERIALIZED_BODY.
 -   Sign SERIALIZED_BODY, base64-encode the result, and store it in `sig`.
+-   Optionally, compute a KEYID and store it in `keyid`.
```

Comment:
Should we be discussing/describing KEYID computation? Is that a future enhancement?

Reply:
I suggest leaving that to #9.

Reply:
Sounds reasonable, I appreciate the change you have made to expand on KEYID.

Reply:
Wow, you're a fast reviewer! :-D

(Marking resolved.)